libs/pilight/protocols/GPIO/ds18b20.c

```diff
@@ -138,16 +135,15 @@ static void *ds18b20Parse(void *param) {
 								}
 								memset(content, '\0', bytes+1);
 
-                                //try to set resolution here - set it temporary to not wear out the EEEPROM
-                                if ((rfd = fopen(ds18b20_w1slave, "w"))) {
-                                    logprintf(LOG_DEBUG, "setting resolution of %s to: %d", ds18b20_w1slave, resolution);
-                                    if (fprintf(ffd, "%d", resolution) < 0 )
-                                        logprintf(LOG_ERR, "cannot set resolution of %s", ds18b20_w1slave);
+								if ((rfd = fopen(ds18b20_w1slave, "w"))) {
+									logprintf(LOG_DEBUG, "setting resolution of %s to: %d", ds18b20_w1slave, resolution);
+									if (fprintf(ffd, "%d", resolution) < 0 )
```

Comment:
Add braces for one-line if statement bodies too

Reply:
The spaces between the `if` and the hooks should also be removed.